json2xml/json2xml.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from typing import Optional, Any
 from xml.dom.minidom import parseString
-from json2xml import dicttoxml
```

Comment:
This was intentionally done as many of our users might already have dicttoxml installed (older version) from pip, so that only the dicttoxml from this codebase will be be used. Please revert this back.

Reply:
Yup. Good catch.